src/ex1/script.js

```diff
@@ -0,0 +1,64 @@
+const listToDo = document.querySelector(".listToDo");
+const count = document.querySelector(".count");
+const buttonClearAll = document.querySelector(".buttonClearAll");
+const newToInput = document.querySelector(".newToInput");
+const buttonAdd = document.querySelector(".buttonAdd");
+
+var removeSVG = '<i class="fa fa-trash"></i>';
+count.textContent = 0;
+
+clickButtonClearAll();
+
+document.addEventListener("keydown", addEnter);
+buttonAdd.addEventListener("click", add);
+buttonClearAll.addEventListener("click", deleteListToDo);
+
+function addEnter(event) {
+  if (
+    event.code === "Enter" &&
+    document.querySelector("#newToDo input").value.length != 0
+  )
+    add();
+}
+
+function deleteListToDo() {
+  if (confirm("Are you sure??") == true) {
+    document.querySelector("#listToDo").innerHTML = "";
+    count.textContent = 0;
+    clickButtonClearAll();
+  }
+}
+
+function clickButtonClearAll() {
```

Comment:
if i had to guess by the function name I would say that this is what happens when you click the clear all, 
I would go with something like setIsActiveClearButton